hello-world/2025_03_08_international_womens_day.md

```diff
@@ -0,0 +1,54 @@
+# A tribute to two women who enabled my career
+
+## My great-grandmother
+My great-grandmother (the mother of the mother of my mother) is without any
+doubt someone who inspired my career, even if she did not live to see the result.
+
+Born in april 1895, she was the only relative I ever knew who was born in the
+19th century. She was the youngest of thirteen children who became a kindergarden
+teacher until she married (she was fired as result, which was common law back
+in the day in the Netherlands, and not only there). 
+
+Her husband died when she was just 32 years old, mother of 3 daughters, in the
+age of 5 (my grandmother), 3, and 1 year old. She rebranded the store she and
+her husband owned, from furniture to yarn (as no men in these days would
+buy furniture from a woman, or take her advice on the subject).
+
+She sent all her daughters to the [MULO](https://en.wikipedia.org/wiki/Meer_Uitgebreid_Lager_Onderwijs)
+(a form of extended primary education). This was definitively frowned up by some,
+as it was not considered useful for women to get educated beyond the bare
+minimum in the 30's of the previous centery. She did it anyway because she
+strongly believed that education was the only way to become financially
+independent.
```

Comment:
```suggestion
She sent all her daughters to the [MULO](https://en.wikipedia.org/wiki/Meer_Uitgebreid_Lager_Onderwijs)
(a form of extended primary education). This was definitively frowned upon by some,
as it was not considered useful for women to get educated beyond the bare
minimum in the 30's of the previous century. She did it anyway because she
strongly believed that education was the only way to become financially
independent.
```